docs/pages/admin-guides/infrastructure-as-code/terraform-provider/ci-or-cloud.mdx

```diff
@@ -48,7 +48,7 @@ GitLab CI, ...)
 ## Prerequisites
 
 You need either:
-- A GCP or AWS VM with terraform installed
+- A GCP or AWS Linux/MacOS VM with Terraform installed
 - A git repo able to run GitHub Actions, GitLab CI or CircleCI jobs
```

Comment:
I don't love the wording of this point. The git repo is not the thing doing CI/CD.